Day_2_Software_engineering_best_practices/05_documentation_git.html

```diff
@@ -0,0 +1,196 @@
+<!--
+
+Part about code style inspired on https://github.com/UWSEDS/LectureNotes/blob/master/11.Style_Doc_Communicate.pptx
+Copyright (c) 2016, UWSEDS (BSD 2-clause "Simplified" License)
+
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Version control with git</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="../img/slides.css">
+  </head>
+  <body>
+    <textarea id="source">
+
+class: center, middle
+
+### Software engineering best practices for scientists
+
+# Version control with `git`
+
+<p style="margin-bottom: 3cm;"></p>
+
+.affiliations[
+  ![:scale 80%](../img/logoUPSayPlusCDS_990.png)
+]
+
+---
+
+# What `git` is used for?
+
+.center[![](../img/git/writing.png)]
+
+---
+
+count: false
+# What `git` is used for?
+
+.center[![](../img/git/coding.png)]
+
+---
+count: false
+# What `git` is used for?
+
+* Keep track of history
+* Safe development of new features
+* Swap between versions
+* **Enable collaborative coding, writing, ...**
+
+---
+
```

Comment:
maybe put here a slide that introduces git -> one system for version control + link